### ISSUE TITLE
ingest ui occasionally blocked by Elixir

### ISSUE THREAD

**Issue by amnonkhen** [ebi-ait/dcp-ingest-central#452]
During the last few weeks ingest-ui suffered a problem with login using Elixir.

Problem scenario

1. go to ingest ui home page
2. click login
3. instead of logging in, it take the user to some elixir screen that says "already registered"

See screenshot:
![image.png](https://images.zenhubusercontent.com/600ed297091d3ea152b83c2d/aeac1b2c-d40f-4e12-92d4-fa45a11476a0)

see also [background on slack](https://embl-ebi-ait.slack.com/archives/CSDN3M2P5/p1631695554098600)

**Comment by aaclan-ebi** [comment]
Duplicate of ebi-ait/dcp-ingest-central#453 Retaining that as that contains more details.
